src/renderers/dom/client/__tests__/ReactMount-test.js

```diff
@@ -20,9 +20,12 @@ describe('ReactMount', function() {
   var WebComponents = undefined;
 
   try {
-    WebComponents = require('WebComponents');
+    if (!/PhantomJS/.test(window.navigator.userAgent)) {
+      WebComponents = require('WebComponents');
+    }
   } catch(e) {
-    /* leave WebComponents undefined */
+    // Parse error expected on super old/sketchy browsers
```

Comment:
Old yes, sketchy no. I would clarify and just say "engines that don't support setters"